Assets/MixedRealityToolkit/_Core/Interfaces/DataProviders/SpatialObservers/IMixedRealitySpatialMeshObserver.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.Core.Definitions.SpatialAwarenessSystem;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Core.Interfaces.DataProviders.SpatialObservers
+{
+    /// <summary>
+    /// The interface contract for Mixed Reality spatial observers.
+    /// </summary>
+    public interface IMixedRealitySpatialMeshObserver : IMixedRealitySpatialObserverDataProvider
```

Comment:
Please re-add the MeshPhysicsLayer property and the MeshPhysicsLayerMask helper.

Reply:
No.